packages/iTwinUI-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -69,7 +69,6 @@ const VirtualizedComboBoxMenu = React.forwardRef(
       () => ({
         minWidth,
         maxWidth: `min(${minWidth * 2}px, 90vw)`,
-        maxHeight: 315,
```

Comment:
I see that this menu is doesn't have `iui-scroll`

Reply:
**[DONE]** Thanks, added `.iui-scroll` to VirtualizedComboBoxMenu